dbms/programs/server/TCPHandler.cpp

```diff
@@ -517,15 +522,13 @@ void TCPHandler::sendHello()
     writeStringBinary(DBMS_NAME, *out);
     writeVarUInt(DBMS_VERSION_MAJOR, *out);
     writeVarUInt(DBMS_VERSION_MINOR, *out);
+    // TODO: If we will broke protocol - add this field:
```

Comment:
Client cannot send its patch version to server.
But server can send its patch version in reply.